frontend/src/layout/panel-layout/PanelLayout.tsx

```diff
@@ -162,8 +162,8 @@ export function PanelLayout({ mainRef }: { mainRef: React.RefObject<HTMLElement>
                     {activePanelIdentifier === 'Shortcuts' && (
                         <ProjectTree root="shortcuts://" searchPlaceholder="Search your shortcuts" />
                     )}
-                    {activePanelIdentifier === 'Data' && (
-                        <ProjectTree root="data://" searchPlaceholder="Search data tools" />
+                    {activePanelIdentifier === 'Data-management' && (
+                        <ProjectTree root="data-management://" searchPlaceholder="Search data tools" />
```

Comment:
Somehow this capitalization feels a bit off. I think something like `DataManagement` would be better. 

As for the protocol itself, whenever we make changes here (and more specifically in `projectTreeDataLogic`), everyone who had made a shortcut to `data://Actions` suddenly gets a broken link. I think it's best we just leave it as `data://` for now.

```suggestion
                    {activePanelIdentifier === 'DataManagement' && (
                        <ProjectTree root="data://" searchPlaceholder="Search data tools" />
```